improver/cli/__init__.py

```diff
@@ -257,19 +257,22 @@ def constrained_inputcubelist_converter(to_convert):
 
 
 @decorator
-def with_output(wrapped, *args, output=None, **kwargs):
+def with_output(wrapped, *args, output=None, no_compression=False, **kwargs):
```

Comment:
I wonder if having `compression_level` option instead wouldn't be better (with `compression_level=0` disabling the compression). It would avoid having negated option name and allow more flexibility (e.g., we could use max compression for acceptance tests to make them smaller).

Reply:
Yes, my thought (when looking at the API) was that this might be a more flexible option to expose.

Reply:
Yes, good point. I agree that would be a better approach and doing it now would save having to create another issue in the future for allowing for the user to set `complevel`.